project/JavalibIRCleaner.scala

```diff
@@ -552,14 +572,26 @@ object JavalibIRCleaner {
     MethodName("fromInt", List(IntRef), ClassRef(JSAny))
   private val jsAnyFromStringMethodName =
     MethodName("fromString", List(ClassRef(BoxedStringClass)), ClassRef(JSAny))
+  private val jsArrayOpsToArrayMethodName =
+    MethodName("jsArrayOps", List(ClassRef(JSArray)), ClassRef(JSArray))
```

Comment:
It's this one:
https://github.com/scala-js/scala-js/blob/33ed7588ad80eaff5802aaf3b6f620b408a200cd/library/src/main/scala-old-collections/scala/scalajs/js/Any.scala#L92-L93
Because `js.ArrayOps` is an `AnyVal(js.Array)`, scalac erases this method to `(js.Array): js.Array)`. It's not a bridge.